docs/api-reference.md

```diff
@@ -483,3 +483,79 @@ type CodeBlock = Omit<CodePaneProps, 'children'> & {
+
+
+### `SlideLayout.VerticalImage`
+
+A layout with one image per slide: a vertical/portrait image with a bulleted list and an optional title.
+[alt text](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt)
+
+| Props               | Type                                                              | Required | Example                                                          |
+|---------------------|-------------------------------------------------------------------|----------|------------------------------------------------------------------|
+| `...slideProps`     | [Slide Props](#slide)                                             | ❌        |                                                                           |
+| `src`               | `string`                                                          | ✅        | `https://raw.githubusercontent.com/FormidableLabs/dogs/main/src/fred.jpg` |
+| `alt`               | `string`                                                          | ✅        | `Fred the dog looking at the camera`                                      |
+| `imgProps`          | `ImgHTMLAttributes<HTMLImageElement>`                             | ❌        | `{ style: { objectFit: 'contain' } }`                                     |
+| `imgContainerProps` | `ComponentProps<typeof FlexBox>`                                  | ❌        | `{ style: { border: '8px solid white' } }`                                |
+| `position`          | `right` | `left`                                                  | ❌        | `right`                                                                   |
+| `title`             | `string | ReactNode`                                              | ❌        | `Fred is a 100% pure bred good boy!`                                      |
+| `titleProps`        | [Text Props](#typography-tags)                                    | ❌        | `{ color: 'red' }`                                                        |
+| `listItems`         | `ReactNode[]`                                                     | ❌        | `['brown hair', 'brown eyes', 'happy boy']`                               |
+| `animateListItems`  | `boolean`                                                         | ❌        | `true`                                                                    |
+| `listType`          | `ordered` | `unordered`                                           | ❌        | `unordered`                                                               |
+| `listProps`         | `ComponentProps<typeof UnorderedList & typeof OrderedList>`       | ❌        | `{ color: 'white' }`                                                      |
+
+
+### `SlideLayout.ThreeUpImage`
+
+A layout with three images per slide: a primary, a top and bottom images.
+[alt text](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt)
```

Comment:
My intention was to be helpful with how to write good alt text since it is still an uncommon practice but we're requiring it here. On second thought, I don't feel it is our responsibility to include that in this way. Removing it.

Reply:
That makes sense. Maybe the description of the `alt` prop would be a good place to put it?

Reply:
That is so much better and seems obvious. I will add that change to https://github.com/FormidableLabs/spectacle/pull/1237.